src/state/sagas/rootSaga.js

```diff
@@ -13,15 +13,30 @@ import authSaga from './auth';
 import powerBISaga from './powerbi';
 
 export default function* rootSaga() {
-  yield all([
-    fork(authSaga),
-    fork(appSaga),
-    fork(organizationSaga),
-    fork(workspaceSaga),
-    fork(solutionSaga),
-    fork(scenarioSaga),
-    fork(scenarioRunSaga),
-    fork(datasetSaga),
-    fork(powerBISaga),
-  ]);
+  const sagas = [
+    authSaga,
+    appSaga,
+    organizationSaga,
+    workspaceSaga,
+    solutionSaga,
+    scenarioSaga,
+    scenarioRunSaga,
+    datasetSaga,
+    powerBISaga,
+  ];
+
+  yield all(
+    sagas.map((saga) =>
+      spawn(function* () {
+        while (true) {
+          try {
+            yield call(saga);
+            break;
+          } catch (e) {
+            console.error(e);
+          }
+        }
+      })
+    )
+  );
```

Comment:
It is the solution recommended by the redux Saga documentation : [https://redux-saga.js.org/docs/advanced/RootSaga](url)

Reply:
Thank you for the link, it was helpful to add some context. :slightly_smiling_face:
Could you add comments above your code, with this link, to document its origin and explain its purpose?

Now, as mentioned in this page:
> [the] rootSaga will terminate on the first error in any [...] saga and crash your whole app
The point of my previous comment was that instead of "hacking" the root saga, it may be a better solution to make sure that the `saveScenario`saga won't throw an error (as we did in other sagas) :smile:

Reply:
I think so it's not a good idea to automaticly apply a fix in the rootsaga level instead of catch properly the error in the saga. it's so likely that 'on error resume next'  lol
As said in documentation : "It is very likely that it makes more sense to let your saga fail in sanely and predictably and handle/log your error."

Reply:
By the way, I found another solution more clean.